apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt_connector.erl

```diff
@@ -508,6 +508,7 @@ connect(Pid, Name) ->
         {ok, _Props} ->
             {ok, Pid};
         {error, Reason} = Error ->
+            log_connect_error_reason(Reason, Name),
```

Comment:
Q: Shouldn't we aim to beautify all the communication that attempts to describes what just happened, not just log messages? E.g. resource start errors and health check statuses that end up in the dashboard?